mlibrary/lib.umich.edu
# Added CallToAction

# Overview
This PR adds a new call-to-action styled link to the website. The design specifications are [outlined in the Figma mockup](https://www.figma.com/design/RNOWJ86DGjXYlfPBAjDvBI/F24-Website-Enhancements?node-id=431-1467&t=AqDNb9ZuSLsV6ACg-0).

> This pull request resolves/closes/fixes [WEBSITE-15](https://mlit.atlassian.net/jira/software/projects/WEBSITE/boards/27?selectedIssue=WEBSITE-15).

The class is applied on the CMS side and imported with the the HTML, similar to the Callout component. The class for the CTA is: umich-lib-cta

You can see three pages that are using the new CTA on deploy here:

- [Request Instruction](https://deploy-preview-324--future-wwwlib-previews.netlify.app/research-and-scholarship/instruction-and-course-support/request-instruction)
- [Undergraduate Research Awards](https://deploy-preview-324--future-wwwlib-previews.netlify.app/research-and-scholarship/awards-and-grants/undergraduate-research-awards)
- [Create a Website](https://deploy-preview-324--future-wwwlib-previews.netlify.app/research-and-scholarship/create-and-design/create-website)

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
  - [x] NVDA VoiceOver
